packages/client/.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 dist
 .DS_Store
+deployments
```

Comment:
we prob don't want to ignore this

Reply:
also having these in the client feels weird? why wouldn't this be alongside all our other solidity/evm files/artifacts?